src/language-css/printer-postcss.js

```diff
@@ -155,7 +164,10 @@ function genericPrint(path, options, print) {
         node.nodes
           ? [
               " {",
-              indent([softline, printSequence(path, options, print)]),
+              indent([
+                node.nodes.length > 0 ? softline : "",
+                printSequence(path, options, print),
+              ]),
               softline,
               "}",
             ]
```

Comment:
```suggestion
          ? [
              " {",
              node.nodes.length > 0 ? indent([
                softline,
                printSequence(path, options, print),
              ]) : "",
              softline,
              "}",
            ]
```

Maybe need reformat.

Reply:
and here
https://github.com/kovsu/prettier/blob/746afeb72b860444494bfa3b3b9959312c743d61/src/language-css/printer-postcss.js#L290-L295

Reply:
Yes.